_data/sites.json

```
@@ -3961,6 +3961,16 @@
         ]
     },
 
+    {
+        "name": "discovery+",
+        "url": "https://support.discoveryplus.com/hc/en-gb/articles/360018959458-How-do-I-delete-my-account-",
+        "difficulty": "hard",
```

```suggestion
        "difficulty": "medium",
```
As for the difficulty, not sure if this web form constitutes a hard difficulty or not, since a user doesn't technically have to contact support directly.